sync2jira/confluence_client.py

```diff
@@ -78,93 +78,96 @@ def update_stat_page(self, confluence_data):
+                'Tags': 10,
+                'FixVersion': 10,
+                'Misc. Fields': 15,
+            }
+            # Use these HTML patterns to search for previous values
+            confluence_html_patterns = {
+                'Created Issues': "Created Issues</td><td>",
+                'Descriptions': "Descriptions</td><td>",
+                'Comments': "Comments</td><td>",
+                'Reporters': "Reporters</td><td>",
+                'Assignees': "Assignees</td><td>",
+                'Status': "Status</td><td>",
+                'Transitions': "Transitions</td><td>",
+                'Title': "Titles</td><td>",
+                'Tags': "Tags</td><td>",
+                'FixVersion': "Fix Version</td><td>",
+                'Misc. Fields': "Misc. Fields</td><td>",
+            }
+            # Update all our data
+            total = 0
+            for topic, html in confluence_html_patterns.items():
+                # Search for previous data
+                try:
+                    ret = re.search(html, page_html)
+                    start_index = ret.span()[1]
+                    new_val = ""
+                    while page_html[start_index] != "<":
+                        new_val += page_html[start_index]
+                        start_index += 1
+                    confluence_data_update[topic] = int(new_val)
+                    total += int(new_val)
+                except AttributeError:
+                    log.warning(f"Confluence failed on parsing {topic}")
+                    total += 0
+                    confluence_data_update[topic] = 0
+
+            # Now add new data
+            for topic in confluence_html_patterns.keys():
+                if topic in confluence_data:
+                    confluence_data_update[topic] += confluence_data[topic]
+                    total += confluence_data[topic]
+            confluence_data_update["Total"] = total
+
+            # Calculate Total Time
+            total_time = 0
+            for topic in confluence_data_times.keys():
+                total_time += confluence_data_update[topic] * confluence_data_times[topic]
+            total_time = datetime.timedelta(seconds=total_time)
+            confluence_data_update["Total Time"] = str(total_time) + " (HR:MIN:SEC)"
+
+            # Build our updated HTML page
+            templateLoader = jinja2.FileSystemLoader(
+                searchpath='usr/local/src/sync2jira/sync2jira/')
+            templateEnv = jinja2.Environment(loader=templateLoader)
+            template = templateEnv.get_template('confluence_stat.jinja')
+            html_text = template.render(confluence_data=confluence_data_update)
+
+            # Finally update our page
+            if html_text.replace(" ", "") != page_html.replace(" ", ""):
+                self.update_page(self.page_id, html_text)
+        except Exception as e:
+            log.error(f"Something went wrong updating confluence!\nException: {e}")
```

Comment:
Use `log.exception(...)`.  The nice thing about it is that it logs the whole traceback, which is useful for debugging.

Just `log.exception("Something went wrong updating confluence!")` should be adequate.